### ISSUE TITLE
tf issues

### ISSUE THREAD

**Issue by natank1** [SciSharp/Keras.NET#74]
When I put in the model this line
Lambda(lambda x: tf.signal.rfft(x, fft_length=[2048]))(x4)

I got an error regarding the tf ("undefined name")

What is the right way to use tensorflow in keras.net

**Comment by deepakkumar1984** [comment]
Can you post the full code?

**Comment by natank1** [comment]
Please:


x0= Input(shape=(12000,))
x00 = Dense(singal_len-1,use_bias=False,trainable=False,weights=[ct],activation="linear")(x0)
x1= Lambda(lambda x: tf.signal.frame(x, frame_length, 240))(x00)
x2= Lambda(lambda x: K.mean(x,axis=-1))(x1)
x2= RepeatVector(frame_length)(x2)
x2 =Permute((2,1))(x2)
x3 = Subtract()([x1,x2])
x4= Dense (frame_length ,use_bias=False,
trainable=False,weights=[ham_diag],activation="linear")(x3)
print (x4.shape)
x5= Lambda(lambda x: tf.signal.rfft(x, fft_length=[2048]))(x4)
x6 =Lambda(lambda  x: scal*K.abs(x))(x5)

m12= Model(inputs=x0, outputs=x6)
